app/Console/Commands/Egg/CheckEggUpdates.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Console\Commands\Egg;
+
+use App\Models\Egg;
+use App\Services\Eggs\Sharing\EggExporterService;
+use Exception;
+use Illuminate\Console\Command;
+
+class CheckEggUpdates extends Command
+{
+    protected $signature = 'p:egg:check-updates';
+
+    public function handle(): void
+    {
+        /** @var EggExporterService $exporterService */
+        $exporterService = app(EggExporterService::class);
+
+        $eggs = Egg::all();
+        foreach ($eggs as $egg) {
+            try {
+                if (is_null($egg->update_url)) {
+                    $this->comment("{$egg->name}: Skipping (no update url set)");
+
+                    continue;
+                }
+
+                $currentJson = json_decode($exporterService->handle($egg->id));
+                unset($currentJson->exported_at);
+
+                $updatedJson = json_decode(file_get_contents($egg->update_url));
```

Comment:
Shouldn't use `file_get_contents`

Reply:
`file_get_contents` is fine, we also use it in the egg importer.